MoviesExample/app/src/main/java/com/novoda/demo/movies/MainActivity.java

```diff
@@ -49,12 +46,8 @@ public void onPageLoadRequested(final int page) {
         });
         resultList.setAdapter(adapter);
 
-        moviesViewModel.moviesLiveData().observe(this, new Observer<MoviesSate>() {
-            @Override
-            public void onChanged(MoviesSate moviesSate) {
-                adapter.setMoviesSate(moviesSate);
-            }
-        });
+        viewDataBinding.setViewmodel(moviesViewModel);
+        viewDataBinding.setLifecycleOwner(this);
```

Comment:
When binding LiveData objects you need to set who's lifecycle is going to be used for observing